packages/keystone/databaseAdapters/ReplicaKnexAdapter.js

```diff
@@ -0,0 +1,117 @@
+const { KnexAdapter } = require('@keystonejs/adapter-knex')
+const { knex } = require('knex')
+const { get, omit } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const logger = getLogger('replicaKnexAdapter')
+
+const MUTABLE_OPERATIONS = ['create', 'update', 'insert', 'delete', 'alter', 'drop', 'truncate', 'rollback', 'del']
```

Comment:
IMHO reverted ops are better (explicitly specify, what to take from replica). And if not sure - go to main db

Reply:
Done that